### PR TITLE
ci(mergify): add a release note check

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: I04c78a95807fd61f66235ba5d2a550eb3c0936b6

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -172,3 +172,19 @@ pull_request_rules:
...
+      - or:
+          - "-title~=^feat"
+          - and:
+              - "title~=^feat"
+              - files~=^releasenotes/notes
+    actions:
+      post_check:
+        title: |
+          {% if check_succeed %}
+          Release notes looks good.
```
Comment (jd) [inline .mergify.yml:187]:
```suggestion
          Release notes found.
```

File: .mergify.yml [hunk 2]
```diff
@@ -172,3 +172,19 @@ pull_request_rules:
...
+          {% else %}
+          A release note need to be added.
```
Comment (jd) [inline .mergify.yml:189]:
```suggestion
          Release note missing.
```

**Review by jd — DISMISSED** [review]
👍

**Comment by sileht** [comment]
@mergify squash

**Pull request merged**